storage/2020-08-04/queue/messages/get.go

```diff
@@ -2,111 +2,99 @@ package messages
 
 import (
 	"context"
+	"fmt"
 	"net/http"
+	"strconv"
 	"strings"
 	"time"
 
-	"github.com/Azure/go-autorest/autorest"
-	"github.com/Azure/go-autorest/autorest/azure"
-	"github.com/Azure/go-autorest/autorest/validation"
-	"github.com/tombuildsstuff/giovanni/storage/internal/endpoints"
+	"github.com/hashicorp/go-azure-sdk/sdk/client"
+	"github.com/hashicorp/go-azure-sdk/sdk/odata"
 )
 
 type GetInput struct {
 	// VisibilityTimeout specifies the new visibility timeout value, in seconds, relative to server time.
 	// The new value must be larger than or equal to 0, and cannot be larger than 7 days.
 	VisibilityTimeout *int
+
+	NumberOfMessages int
```

Comment:
```suggestion
	// NumberOfMessages specifies the (maximum) number of messages that should be retrieved from the queue.
	// This can be a maximum of 32.
	NumberOfMessages int
```